ui/src/components/Header.js

```diff
@@ -23,7 +28,7 @@ const styles = {
   },
   toggleButtonGroup: {
     borderRadius: "20px",
-    // Must be same color as header background to fix border-radius bleed
+    // Must be same color as appBar background to fix border-radius bleed
     // (https://stackoverflow.com/a/30356787/6447189)
     backgroundColor: "#55a146"
```

Comment:
I think in general we should have constants somewhere for values like this. Would be easy to keep track of changes too.

Reply:
Unfortunately the appBar background is a gradient. I used the midpoint color here. Added a comment.